Tests/EventsExporter/EventsExporterTests.swift

```diff
@@ -19,7 +19,7 @@ class EventsExporterTests: XCTestCase {
     }
 
     func testWhenExportSpanIsCalled_thenTraceAndLogsAreUploaded() throws {
-        try XCTSkipIf(true)
+        //try XCTSkipIf(true)
```

Comment:
Are you planning to leave the code commented here or are you trying something?

Reply:
Forgot to remove comments. They were disabled, I enabled them and forgot to cleanup.